### PR TITLE
Added 4 programs in new folder assignment3

### PR DESCRIPTION
**Pull request by hryxna**
Hello @chandrakant100, I have added 4 new Programs namely:
1.Palindrome - to check whether the given string is palindrome or not.
2.Anagrams - to check whether the given string is an anagram or not
3.StringReverse - it has different methods by which you can reverse a string.
4.Stack Max - to find out max-value among 2 stacks.

Hope So you review and find them useful.
Have a good day 😄.

### CODE REVIEW

**Comment by chandrakant100** [comment]
> Hello @chandrakant100, I have added 4 new Programs namely:
> 1.Palindrome - to check whether the given string is palindrome or not.
> 2.Anagrams - to check whether the given string is an anagram or not
> 3.StringReverse - it has different methods by which you can reverse a string.
> 4.Stack Max - to find out max-value among 2 stacks.
> 
> Hope So you review and find them useful.
> Have a good day 😄.

Good job!!

File: assignment3/Anagrams.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (chandrakant100) [inline assignment3/Anagrams.java:1]: Add some comments to this program describing the steps in bullet points

**Review by chandrakant100 — CHANGES_REQUESTED** [review]

**Review by hryxna — COMMENTED** [review]
Thanks for your review
I have added the needed description and bullets to the Anagrams.java

**Review by chandrakant100 — APPROVED** [review]

**Comment by chandrakant100** [comment]
> Thanks for your review
> I have added the needed description and bullets to the Anagrams.java

Good job!!

**Comment by hryxna** [comment]
Thank you for accepting my contribution 😄.

**Pull request merged**